lib/mavenlink/model.rb

```diff
@@ -120,17 +120,18 @@ def self.update_attributes
 
     # @param source_record [Brainstem::Record, nil]
     # @param client [Mavenlink::Client]
-    def self.wrap(source_record = nil, client = Mavenlink.client)
-      new({}, source_record, client)
+    def self.wrap(source_record = nil, client = Mavenlink.client, scope = {})
```

Comment:
Can you document the scope for all places/methods that it was added (with `@param`)